app/Models/Training/WaitingList/WaitingListAccount.php

```diff
@@ -176,6 +178,21 @@ public function getEligibilityAttribute()
         return $this->atcHourCheck() && $this->allFlagsChecker() && $this->current_status->name == 'Active';
     }
 
+    public function getTheoryExamPassedAttribute(): bool
+    {
+        if ($this->waitingList->department === WaitingList::PILOT_DEPARTMENT || ! $this->waitingList->cts_theory_exam_level) {
+            return false;
+        }
+
+        // the internal CTS member column id not the same as the account id
+        // so search on the indexed column cid.
+        $memberId = Member::where('cid', $this->account_id)->first()->id;
+
+        return TheoryResult::where('student_id', $memberId)
```

Comment:
Not sure how much worth it would be in this situation, but we could do this query through a relation on the Cts/Member model with scopes for pass etc

Reply:
Only concern I had there is the query might not use the student_id first which isn't a PK but an indexed column

Reply:
I feel like this would be better as a TheoryResult::forAccount(1258635); and abstract the knowledge of it being member_id / cid to the model?

Reply:
Good shout, have moved it to the model on a static method, only performing the first part of the query. 
https://github.com/VATSIM-UK/core/pull/2824/commits/b08f888d7de73b38df763253a77378706fb73835